crates/sargon/src/profile/v100/address/non_fungible_local_id.rs

```diff
@@ -43,6 +43,33 @@ impl NonFungibleLocalId {
         NonEmptyMax64Bytes::try_from(bytes.as_ref())
             .map(|value| Self::Bytes { value })
     }
+
+    pub fn as_bytes(&self) -> Option<NonEmptyMax64Bytes> {
+        if let NonFungibleLocalId::Bytes { value } = self {
```

Comment:
prefer `EnumAsInner` on `NonFungibleLocalId` then this method should have been generated for us

Reply:
temporary, will use Michael's code once once pushed